actions/salesforce/README.md

```diff
@@ -0,0 +1,58 @@
+# Salesforce
+
+Action Package interacting with Salesforce resources.
+
+Possible actions with this package are:
+
+- query the Salesforce resources in an SQL like manner
+
+## Prompts
+
+```
+What are the latest 3 contacts created?
+```
+
+> Here are the last 3 contacts created:
+>
+> 1. Rose Gonzalez - Created on 2024-10-24
+> 2. Sean Forbes - Created on 2024-10-24
+> 3. Jack Rogers - Created on 2024-10-24
+
+
+```
+What are the biggest 3 deals closed this month?
+```
+
+> Here are the biggest deals closed this month:
+>
+> 1. Grand Hotels Generator Installations - Amount: $350,000, Closed on 2024-10-03
+> 2. Grand Hotels Emergency Generators - Amount: $210,000, Closed on 2024-10-01
+> 3. United Oil SLA - Amount: $120,000, Closed on 2024-10-12
+
+## Authorization
+
+This action package uses OAuth2 client credentials flow to authenticate with Salesforce.
+
+To authenticate with Salesforce, you need to create an external connected app in Salesforce and provide the following information:
+
+- client_id (Consumer Key)
+- client_secret (Consumer Secret)
+- domain (e.g.  `https://robocorp-dev-ed.develop.my.salesforce.com`)
```

Comment:
I would call this "My Salesforce API endpoint" instead of "domain". If this would be only "domain" then value passed as secret would be only the "robocorp-dev-ed.develop.my" part.

Reply:
I'll call this domain url. This is not specific to api only.